packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -12,7 +12,7 @@
 
 %if %{dnf_install}
 %global python_libdir %{python3_sitelib}
-%global plugins_dir %{python3_sitelib}/dnf_plugins
+%global plugins_dir %{python3_sitelib}/dnf-plugins
```

Comment:
this is in line with the old spec of 3.5.7:

https://github.com/theforeman/foreman-packaging/blob/f1ce67c948d47772d769417e753fa2dd0fe411f7/packages/client/katello-host-tools/katello-host-tools.spec#L152